notebook.ipynb

```diff
@@ -7,10 +7,10 @@
     "cell_id": "8c3fc4c1-e045-41fd-9fa9-4068b4b87a78",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df['children'].hist(); # builds and shows a histogram for the number of children by per observation
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graceuche/northeastern-final-project/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>